packages/bytecode-utils/src/lib/bytecode.spec.ts

```diff
@@ -19,22 +18,24 @@ const BYTECODE_EXPERIMENTAL = readFileSync(
 const BYTECODE_WITHOUT0X = readFileSync(
   `${BYTECODES_FOLDER}/without0x.hex`
 ).toString();
+const BYTECODE_WITHOUTAUXDATA = readFileSync(
+  `${BYTECODES_FOLDER}/withoutauxdata.hex`
+).toString();
 
-test("get contract's bytecode by address and provider", async (t) => {
-  const ethereumProvider = provider('https://eth-mainnet.public.blastapi.io');
-  t.is(
-    await get('0x00000000219ab540356cBB839Cbe05303d7705Fa', ethereumProvider),
-    BYTECODE_IPFS
-  );
+test('return bytecode without auxdata while splitting bytecode into execution bytecode and auxadata', (t) => {
```

Comment:
Implies the returned bytecode will be return excluding the auxdata. Didn't get it first.
```suggestion
test('return the whole bytecode when the bytecode that doesn't contain auxdata', (t) => {
```